packages/osd-plugin-helpers/README.md

```diff
@@ -37,37 +38,44 @@ $ plugin-helpers help
   Usage: plugin-helpers [command] [options]
 
   Commands:
-      build
-        Copies files from the source into a zip archive that can be distributed for
-        installation into production OpenSearch Dashboards installs. The archive includes the non-
-        development npm dependencies and builds itself using raw files in the source
-        directory so make sure they are clean/up to date. The resulting archive can
-        be found at:
+    build
+      Copies files from the source into a zip archive that can be distributed for
+      installation into production OpenSearch Dashboards installs. The archive includes the non-
+      development npm dependencies and builds itself using raw files in the source
+      directory so make sure they are clean/up to date. The resulting archive can
+      be found at:
+
+        build/{plugin.id}-{opensearchDashboardsVersion}.zip
+
+      Options:
+        --skip-archive                       Don't create the zip file, just create the build/opensearch-dashboards directory
+        --opensearch-dashboards-version, -v  OpenSearch Dashboards version that the
+     
+    version
+      Updates the version to a provided parameter, or syncs it with the version of OpenSearch Dashboards
   
-          build/{plugin.id}-{opensearchDashboardsVersion}.zip
-  
-        Options:
-          --skip-archive        Don't create the zip file, just create the build/opensearch-dashboards directory
-          --opensearch-dashboards-version, -v  OpenSearch Dashboards version that the
-    
+      Options:
+        --sync        Update the versions to match Dashboards' (default)
+        --set         Update the version to a specific value
```

Comment:
Can we provide an example that shows the version formatting expected?

Reply:
Updated the string to call out that a "semantic version" is expected